---
layout: default
title: Authors, References and Citation
nav_order: 7
---

# Authors, References and Citation
{:.no_toc }
Here you will find all the useful metadata about people and dependencies linked to AMPLIPIPER
{:.fs-6.fw-300 }

<details open markdown="block">
  <summary>
    Table of contents
  </summary>
  {:.text-delta }
- TOC
{:toc}
</details>

## Developers 

* [Martin Kapun](https://github.com/capoony), NHM Wien
* [Sonja Steindl](https://github.com/sonjastndl), NHM Wien
* [Astra Bertelli](https://astrabert.vercel.app), University of Pavia
* [Stefan Eichert](https://github.com/stefaneichert), NHM Wien

Other people?? Order??

## Contributors

* [Luise Kruckenhauser](https://www.nhm-wien.ac.at/en/luise_kruckenhauser)
* [Sandra Kirchner](https://www.nhm-wien.ac.at/en/sandra_kirchner)
* [Nikolaus Szucsich](https://www.nhm-wien.ac.at/en/nikolaus_szucsich)
* [Elisabeth Haring](https://www.nhm-wien.ac.at/en/elisabeth_haring)

Other people?? Order??

## References

**Software**

* **NanoFilt**: De Coster, W., D'Hert, S., Schultz, D. T., Cruts, M., & Van Broeckhoven, C. (2018). NanoPack: visualizing and processing long-read sequencing data. Bioinformatics, 34(11), 1933-1934. doi: 10.1093/bioinformatics/bty102
* **NanoCaller**: Ahsan, M.U., Liu, Q., Fang, L. et al. NanoCaller for accurate detection of SNPs and indels in difficult-to-map regions from long-read sequencing by haplotype-aware deep neural networks. Genome Biol 22, 261 (2021). https://doi.org/10.1186/s13059-021-02472-2.
* **Pigz**: Adler, M., https://github.com/madler/pigz
* **Samtools**: Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N.,... & 1000 Genome Project Data Processing Subgroup. (2009). The Sequence Alignment/Map format and SAMtools. Bioinformatics, 25(16), 2078-2079. doi: 10.1093/bioinformatics/btp352
* **Minimap**: Li, H. (2018). Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics, 34(18), 3094-3100. doi: 10.1093/bioinformatics/bty191
* **BCFtools**: Danecek P, Bonfield JK, et al. Twelve years of SAMtools and BCFtools. Gigascience (2021) 10(2):giab008 link
* **MAFFT**: Katoh, K., & Standley, D. M. (2013). MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Molecular Biology and Evolution, 30(4), 772-780. doi: 10.1093/molbev/mst010
* **IQtree**: Nguyen, L. T., Schmidt, H. A., von Haeseler, A., & Minh, B. Q. (2015). IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies. Molecular Biology and Evolution, 32(1), 268-274. doi: 10.1093/molbev/msu300
* **Astral**: Zhang, C., Rabiee, M., Sayyari, E., & Mirarab, S. (2018). ASTRAL-III: polynomial time species tree reconstruction from partially resolved gene trees. BMC Bioinformatics, 19(1), 153. doi: 10.1186/s12859-018-2129-4
* **ASAP**: Puillandre N, Brouillet S, Achaz G. ASAP: assemble species by automatic partitioning. Mol Ecol Resour. 2021; 21: 609–620. https://doi.org/10.1111/1755-0998.13281
* **GNU Parallel**: Tange, O. (2011). GNU Parallel - The Command-Line Power Tool. ;login: The USENIX Magazine, 36(1), 42-47.
* **BLAST**: Altschul, S. F., Gish, W., Miller, W., Myers, E. W., & Lipman, D. J. (1990). Basic local alignment search tool. Journal of Molecular Biology, 215(3), 403-410. doi: 10.1016/S0022-2836(05)80360-2

**Python Packages**

* **Biopython**: Cock, P. J. A., Antao, T., Chang, J. T., Chapman, B. A., Cox, C. J., Dalke, A.,... & de Hoon, M. J. L. (2009). Biopython: freely available Python tools for computational molecular biology and bioinformatics. Bioinformatics, 25(11), 1422-1423. doi: 10.1093/bioinformatics/btp163
* **Edlib**: Šošić, M., & Šikić, M. (2017). Edlib: a Python library for fast and efficient edit distance calculation. Bioinformatics, 33(11), 1741-1742. doi: 10.1093/bioinformatics/btx053
* **CairoSVG**: Kozea, https://github.com/Kozea/CairoSVG
* **PyMSAviz**: https://github.com/moshi4, https://github.com/moshi4/pyMSAviz
* **ETE3**: Huerta-Cepas, J., Serra, F., & Bork, P. (2016). ETE 3: reconstruction, analysis, and visualization of phylogenomic data. Molecular Biology and Evolution, 33(6), 1635-1638. doi: 10.1093/molbev/msw046
* **Pandas**: McKinney, W. (2010). Data structures for statistical computing in Python. Proceedings of the 9th Python in Science Conference, 51-56.
* **Matplotlib**: Hunter, J. D. (2007). Matplotlib: A 2D graphics environment. Computing in Science & Engineering, 9(3), 90-95. doi: 10.1109/MCSE.2007.55
* **BeautifulSoup**: Richardson, L. (2007). Beautiful Soup. Crummy, 1-12.
* **SciPy**: Virtanen, P., Gommers, R., Oliphant, T. E., Haberland, M., Reddy, T., Cournapeau, D.,... & SciPy 1.0 Contributors. (2020). SciPy 1.0: Fundamental Algorithms for Scientific Computing. arXiv preprint arXiv:2006.12345.

**R Packages**

* **R-base**: R Core Team. (2021). R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing.
* **Bioconductor-ggtree**: Yu, G., Smith, D. K., Zhu, H., Guan, Y., & Lam, T. T. Y. (2017). ggtree: an R package for visualization and annotation of phylogenetic trees with their covariates and other associated data. Methods in Ecology and Evolution, 8(1), 28-36. doi: 10.1111/2041-210X.12628
* **R-tidyverse**: Wickham, H., Averick, M., Bryan, J., Chang, W., McGowan, L. D., François, R.,... & Yutani, H. (2019). Welcome to the tidyverse. Journal of Open Source Software, 4(43), 1686. doi: 10.21105/joss.01686
* **Bioconductor-treeio**: Li, M., & Ma, J. (2019). treeio: an R package for phylogenetic tree input/output and manipulation. Bioinformatics, 35(11), 1939-1941. doi: 10.1093/bioinformatics/btz024
* **R-phytools**: Revell, L. J. (2012). phytools: an R package for phylogenetic comparative biology (and other things). Methods in Ecology and Evolution, 3(2), 217-223. doi: 10.1111/j.2041-210X.2011.00169.x
* **R-ape**: Paradis, E., Claude, J., & Strimmer, K. (2004). APE: analyses of phylogenetics and evolution in R language. Bioinformatics, 20(2), 289-290. doi: 10.1093/bioinformatics/btg412
* **R-treedist**: FitzJohn, R. G. (2012). Diversitree: comparative phylogenetic analyses of diversification in R. Methods in Ecology and Evolution, 3(6), 1084-1092. doi: 10.1111/j.2041-210X.2012.00234.x
* **R-reshape2**: Wickham, H. (2007). Reshaping data with the reshape package. Journal of Statistical Software, 21(12), 1-20. doi: 10.18637/jss.v021.i12

## Citation

When using AMPLIPIPER, please consider reporting the following citation: 

**TBD**